packages/entando-apimanager/modules/api/apiManager.js

```diff
@@ -168,19 +167,9 @@ export const makeRealRequest = (request, page) => {
       }
     }
     return response;
-  }).catch((e) => {
-    store.dispatch(addToast(
-      formattedText(
-        `app.${normalizeErrorMessage(e.message)}`,
-        null,
-        { domain: getDomain(store.getState()) },
-      ),
-      TOAST_ERROR,
-    ));
-    // eslint-disable-next-line no-console
-    console.error(e);
-    throw e;
-  });
+  }).catch(e => (
+    Promise.reject(new Error(`app.${normalizeErrorMessage(e.message)}`))
```

Comment:
The info about domain is lost here. Do we think is useless?

Reply:
it was supposed to use only info from `process.env.DOMAIN` in app builder but anyway I have implemented back the `domain` but this time injected in `error.data`.

Reply:
Doing this way, we suppose on the other side someone should know the Error object has this added data field.
What about passing a custom error containing a message descriptor with message, defaultMessage and parameters? @nicpuddu

Reply:
sounds like a good idea @gidesan

Reply:
okay guys, I'll make a custom error for this then ;)

Reply:
@jeffgo10 I'd just like to add that to me it is way more sensible if the `apimanager` exports among other things a json compliant with the app-builder i18n so that we are not forced to implement those translations from scratch.

see the locales files inside app-builder itself.

Reply:
@nicpuddu sorry i'm not sure if I get what you mean, but do you meant not to provide defaultmessage for the Errors inside apimanager or we provide a json format locale library for the message descriptors inside apimanager? What I just did for now is just to decamelize the message ids and make them as default messages. Let me know what you guys think

Reply:
the module itself should just be exporting a message file akin to the ones in app builder, so that appbuilder can just import them and merge it with theirs instead of having to deal with resetting the messages on its own.

just like we used to do for plugins.

Reply:
okay @nicpuddu. Thanks for the suggestion. I'll work on that.

Reply:
Hi @nicpuddu. I have added `locales` as an export and also used it as default message for the `ErrorI18n` error dispatch. Please review

Reply:
@jeffgo10 can you please capitalize the first letter of the translated messages? :)

Reply:
@gidesan done :)